template/src/App.tsx

```diff
@@ -21,7 +21,7 @@ const App = (): ReactElement | null => {
 
   const renderNavigation = (): ReactElement => {
     // replace with actual user profile data once loaded, or fallback to guest nav
-    const currentUser = false;
+    const currentUser = true;
```

Comment:
I'm assuming neither of these matter as they are meant to be replaced, but does `AppNav` render properly without a real user?

Reply:
Correct